servers/web/main.py

```diff
@@ -0,0 +1,14 @@
+from fastapi import FastAPI
+
+app = FastAPI()
+
+
+@app.get("/")
+def read_root():
+    return {"Hello": "World"}
```

Comment:
Add description to endpoints
@app.get("/", summary="Root route", description="Returns Hello World")
def read_root():
    return {"Hello": "World"}